catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorObjectListTest.java

```diff
@@ -0,0 +1,163 @@
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static org.catrobat.catroid.uiespresso.content.brick.utils.BrickDataInteractionWrapper.onBrickAtPosition;
+import static org.catrobat.catroid.uiespresso.util.UiTestUtils.getResources;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.core.Is.is;
+import static org.junit.runners.Parameterized.Parameter;
+
+import java.util.Arrays;
+
+import static android.support.test.espresso.Espresso.onData;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.scrollTo;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.hasSibling;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+@Category({Cat.CatrobatLanguage.class, Level.Smoke.class})
+@RunWith(Parameterized.class)
+public class FormulaEditorObjectListTest {
+
+	String nameSprite2 = "testSprite2";
+
+	@Rule
+	public BaseActivityInstrumentationRule<ScriptActivity> baseActivityTestRule = new BaseActivityInstrumentationRule<>(ScriptActivity.class, true, false);
+
+	@Parameterized.Parameters(name = "{1}" + "-Test")
+	public static Iterable<Object[]> data() {
+		return Arrays.asList(new Object[][] {
+				{R.string.formula_editor_object_transparency, "transparency"},
+				{R.string.formula_editor_object_brightness, "brightness"},
+				{R.string.formula_editor_object_color, "color"},
+				{R.string.formula_editor_object_background_number, "look_number"},
+				{R.string.formula_editor_object_background_name, "look_name"},
+				{R.string.formula_editor_object_x, "position_x"},
+				{R.string.formula_editor_object_y, "position_y"},
+				{R.string.formula_editor_object_size, "size"},
+				{R.string.formula_editor_object_rotation, "direction"},
+				{R.string.formula_editor_object_layer, "layer"},
+				{R.string.formula_editor_function_collides_with_edge, "touches_edge"},
+				{R.string.formula_editor_function_collision, "touches_object"},
+				{R.string.formula_editor_function_touched, "touches_finger"},
+				{R.string.formula_editor_object_x_velocity, "x_velocity"},
+				{R.string.formula_editor_object_y_velocity, "y_velocity"},
+				{R.string.formula_editor_object_angular_velocity, "angular_velocity"}
+		});
+	}
+
+	@Parameter
+	public @StringRes
+	int formulaEditorObjectFunction;
+
+	@Parameter(1)
+	public String testName;
+
+	@Before
+	public void setUp() throws Exception {
+		Project project = new Project(null, "FormulaEditorObjectTest");
```

Comment:
please put this in a create project method, or use an existing createProject method.